src/types.js

```diff
@@ -58,5 +58,13 @@ export const TYPES = {
   inAppPurchasesPolicy: {
     name: 'In App Purchases Policy',
     fileName: 'in_app_purchases_policy'
+  },
+  userConsentPolicy: {
+    name: 'User Consent Policy',
+    fileName: 'user_consent_policy'
+  },
+  servicesList: {
+    name: 'List of Services',
+    fileName: 'services_list'
```

Comment:
This seems to be a description of the services on which terms apply, and until which point they can be applied.

I am not sure it should be tracked in a specific service: shouldn't this interpretation document rather be used to assess whether some document should be tracked or not depending on the service?

For example, I understand with this document that the “AdMob” service should track the “Google Ads Controller-Controller Data Protection Terms”.